extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -46,7 +47,9 @@
 }
 
 .i-amphtml-story-interactive-mid-selection .i-amphtml-story-interactive-slider-bubble {
-  transform: translate(-50%, -160%) scale(1) !important;
+  --scale-var: calc(var(--fraction) + 1) !important;
+  /* 70.7% is the sqrt of 2 over 2 */
```

Comment:
We can explain this better: `/* Move the bubble up by sqrt(2)/2=70.7% to account for the pointy corner */`